js/adapt-extension.js

```diff
@@ -1,6 +1,6 @@
 import Adapt from 'core/js/adapt';
 import Backbone from 'backbone';
-import ExtensionView from 'extensions/adapt-extension/js/adapt-extensionView';
+import ExtensionView from 'extensions/adapt-extension/js/ExtensionView.js';
```

Comment:
```suggestion
import ExtensionView from 'extensions/adapt-extension/js/ExtensionView';
```

don't need the `.js` here.